pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -251,21 +272,26 @@ def connection_select(context, name):
                      default='all'))
 @add_options(verify_option)
 @click.pass_obj
-def connection_add(context, **options):
+def connection_add(context, name, **options):
     """
     Add a new WBEM connection definition from specified options.
 
     Create a new WBEM connection definition in the connections file from
     the specified options. A connection definition with that name must not
     yet exist.
 
-    The current connection remains unchanged by this command.
+    The NAME argument MUST exist. It define the server uri
```

Comment:
To me it does not sound OK to talk about "existing" options. Suggest to change "The NAME argument MUST exist." to "The NAME argument is required.".

Reply:
Agreed and dropped next sentence

Reply:
DONE